src/networks.ts

```diff
@@ -65,6 +66,7 @@ export const PRODUCTION_NETWORKS: { [chainId: number]: PublicNetwork } = {
   },
   [CHAIN_IDs.ARBITRUM]: {
     name: "Arbitrum One",
+    family: NONE,
```

Comment:
This could potentially also be ORBIT, but since it wasn't previously defined, NONE will maintain the status quo.